examples/onnx/yolov5-videos/cortex.yaml

```diff
@@ -0,0 +1,13 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
+
+- name: youtube-yolov5
+  kind: SyncAPI
+  predictor:
+    type: onnx
+    path: predictor.py
+    model_path: yolov5s.onnx
```

Comment:
I've uploaded it to `s3://cortex-examples/onnx/yolov5-youtube/yolov5s.onnx` for now, and I can move it depending on what we decide regarding the API name